deployments/docker/generate-config.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+
+import os
+from jinja2 import Environment, FileSystemLoader
+
+def env_override(default_value, key):
+    """ Gets value of the environment variable provided in the key """
+    return os.getenv(key, default_value)
+
+def generate_config(path_to_config_templates, path_to_new_config, curr_service):
+    # Load templates into Jinja environment
+    env = Environment(loader = FileSystemLoader(path_to_config_templates), trim_blocks=True, lstrip_blocks=True)
+    env.filters['env_override'] = env_override
+
+    # Renders config templates for each service into a config.yaml file
+    template = env.get_template(curr_service + '_tmpl.j2')
+    print(template.render())
+
+    file = open(path_to_new_config, 'w')
+    file.write(template.render())
+    file.close()
+
+# TODO Fix relative directory
+services = ["provisioning", "verification", "vts"]
+templates = "./config-templates"
+for service in services:
```

Comment:
I meant something different, sorry for not being very clear: I wanted this script to be generally useful, even outside the docker build.  But it doesn't matter, it's ok as it is.

Reply:
Ah apologies, I must have misunderstood 😅